locations/spiders/sos_br.py

```diff
@@ -0,0 +1,39 @@
+from typing import Any, Iterable
+
+from scrapy import FormRequest, Spider
+from scrapy.http import Response
+
+from locations.categories import apply_category
+from locations.items import Feature
+
+
+class SosBRSpider(Spider):
+    name = "sos_br"
+    item_attributes = {"brand": "SOS Technologia e Educação", "brand_wikidata": "Q129847471"}
+    requires_proxy = True
+
+    def start_requests(self) -> Iterable[FormRequest]:
+        yield FormRequest(
+            url="https://www.sos.com.br/BuscaUnidadesMapa",
+            method="POST",
+            formdata={"isLead": "false", "isMapa": "true", "isTeste": "false"},
+        )
+
+    def parse(self, response: Response, **kwargs: Any) -> Any:
+        for location in response.json()["Unidades"]:
+            item = Feature()
+            item["extras"]["sources"] = [location]
+            item["ref"] = location["ID"]
+            item["name"] = location["NomeFantasia"]
```

Comment:
Worth adding anything to dict_parser for non english name variants?

Reply:
I don't know. I feel like it does not happen a ton and could lead to other stuff getting mapped that should not.